nets/DevNetSpec.ts

```diff
@@ -19,11 +20,13 @@ export abstract class DevNetSpec extends NetSpec {
   readonly binary
   readonly chain
   readonly nodeCount
+  readonly #rawChainSpecPaths
```

Comment:
Can you initialize this above `rawChainSpecPath`, and use `PermanentMemo` from `util`?